.prettierrc

```diff
@@ -0,0 +1,17 @@
+{
+  "parser": "babel",
+  "printWidth": 80,
+  "tabWidth": 2,
+  "useTabs": false,
+  "semi": true,
+  "singleQuote": true,
+  "jsxSingleQuote": false,
+  "trailingComma": "none",
```

Comment:
You do not like trailing commas? I do. :)

Reply:
I have no strong opinion about them. Currently our setup of prettier was this, but we can surely sure change it...

Reply:
I would prefer `es5` value. I think it creates much less error prone environment (as we do not constantly forget writing commas when appending items to objects).

Reply:
Ok